src/DssSpell.sol

```diff
@@ -39,72 +39,67 @@ contract DssSpellAction is DssAction {
-        DssExecLib.setIlkStabilityFee("WBTC-B", SEVEN_PT_ONE_EIGHT_PCT_RATE, true);
+        // ---------- Whistleblower Bounty Payment ----------
+        // Forum: https://forum.makerdao.com/t/ad-derecognition-due-to-operational-security-breach-02-02-2024/23619/10
+        // Note: payments are skipped on goerli
 
-        // Decrease the WBTC-C Stability Fee (SF) by 0.02 percentage points, from 6.45% to 6.43%.
-        DssExecLib.setIlkStabilityFee("WBTC-C", SIX_PT_FOUR_THREE_PCT_RATE, true);
+        // Transfer 20.84 MKR to whistelblower at 0xCDDd2A697d472d1e8a0B1B188646c756d097b058
 
-        // ---------- Spark Protocol DC-IAM Parameter Changes (main spell) ----------
-        // Forum: https://forum.makerdao.com/t/feb-9-2024-proposed-changes-to-sparklend-for-upcoming-spell/23656
-        // Poll: https://vote.makerdao.com/polling/QmS8ch8r
 
-        // Increase the DIRECT-SPARK-DAI Maximum Debt Ceiling (line) by 300 million DAI from 1.2 billion DAI to 1.5 billion DAI.
-        // Increase the DIRECT-SPARK-DAI Target Available Debt (gap) by 20 million DAI from 20 million DAI to 40 million DAI.
-        // Increase the DIRECT-SPARK-DAI Ceiling Increase Cooldown (ttl) by 12 hours from 12 hours to 24 hours.
-        DssExecLib.setIlkAutoLineParameters("DIRECT-SPARK-DAI", /* line = */ 1500 * MILLION, /* gap = */ 40 * MILLION, /* ttl = */ 24 hours);
+        // ---------- WBTC vault gap Changes ----------
+        // Forum: https://forum.makerdao.com/t/stability-scope-parameter-changes-10-wbtc-a-c-dc-iam-gap/23765
 
-        // Increase the DIRECT-SPARK-DAI buffer by 20 million DAI from 30 million DAI to 50 million DAI.
-        DssExecLib.setValue(DIRECT_SPARK_DAI_PLAN, "buffer", 50 * MILLION * WAD); // NOTE: adjusting DIRECT_SPARK_DAI_PLAN.buffer value to WAD
+        // Increase the WBTC-A gap by 2 million DAI from 2 million DAI to 4 million DAI
+        DssExecLib.setIlkAutoLineParameters("WBTC-A", /* line = */ 500 * MILLION, /* gap = */ 4 * MILLION, /* ttl = */ 24 hours);
 
-        // ---------- Push USDP out of input conduit ----------
-        // Forum: https://forum.makerdao.com/t/proposed-housekeeping-items-upcoming-executive-spell-2024-02-22/23697
-        // Note: skipping since there is no Jar for USDP on Goerli
+        // Increase the WBTC-C gap by 6 million DAI from 2 million DAI to 8 million DAI
+        DssExecLib.setIlkAutoLineParameters("WBTC-C", /* line = */ 500 * MILLION, /* gap = */ 8 * MILLION, /* ttl = */ 24 hours);
 
-        // ---------- yank vest streams ----------
-        // Forum: https://forum.makerdao.com/t/proposed-housekeeping-items-upcoming-executive-spell-2024-02-22/23697
-        // Note: payments are skipped on goerli
 
-        // ---------- Trigger Spark Proxy Spell ----------
-        // Forum: https://forum.makerdao.com/t/feb-14-2024-proposed-changes-to-sparklend-for-upcoming-spell/23684
+        // ---------- Spark Proxy Spell ----------
```

Comment:
Do you want to add forum links (since they are ready in the sheet)? I would say it's up to you

Reply:
Yeah, will add it in 30mins